Automattic/jetpack
# Added is_automattician check for Atomic sites

Our goal is to help us quickly find a way to gate features for a11n on Atomic sites.

## Proposed changes:

* Added `is_automattician_feature_flags_only` on Jetpack sites as an alternative to `is_automattician` present on simple sites.
* Note: It won't work outside the Atomic environment.

### Other information:


## Does this pull request change what data or activity we track or use?

No.

## Jetpack product discussion
Slack thread: p1703196693161289-slack-CDLH4C1UZ

## Testing instructions:






* Apply this code inside an Atomic site using the Jetpack beta tester.
* Edit a file like `/htdocs/wp-content/plugins/jetpack-dev/modules/comments.php` by adding `echo '( new Visitor() )->is_automattician_feature_flags_only: ' . ( new \Automattic\Jetpack\Status\Visitor() )->is_automattician_feature_flags_only() . "\n";` to it.
* When Proxied, it should return true and false otherwise.
